docs/commcare-cloud/commands/index.md

```diff
@@ -277,9 +277,9 @@ authenticate using the pem file (or prompt for root password if there is no pem
   --list-hosts          outputs a list of matching hosts; does not execute
                         anything else
   -M MODULE_PATH, --module-path=MODULE_PATH
-                        prepend colon-separated path(s) to module library
-                        (default=[u'~/.ansible/plugins/modules',
-                        u'/usr/share/ansible/plugins/modules'])
+                        prepend colon-separated path(s) to module library (def
+                        ault=[u'~/.ansible/plugins/mo
+                        dules', u'/usr/share/ansible/plugins/modules'])
```

Comment:
how come these changed?

Reply:
Oops, reverted this.

If you're curious why it happened, my `make` command generated a different linebreak than what was was on master, which danny fixed here: https://github.com/dimagi/commcare-cloud/commit/5b02b4cc8de1737a6fe6183cef06419713e2b9b9